app/models/training/question.rb

```diff
@@ -4,5 +4,55 @@ class Question < Content
     def self.content_type_id
       'question'
     end
+
+    def options
+      answers.map do |option|
+        OpenStruct.new(id: option.keys[0].to_s.to_i, label: option.values[0][0], correct?: option.values[0][1])
```

Comment:
Did you want to use the dry-struct approach with validation/coercion in a later PR? (which validates the JSON written by the content team)

Reply:
Yes, but I was hoping to leave that up to you - my first efforts were dismal.

Reply:
I will add a draft struct to the [content checking code](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/566/files#diff-ce6ef186511b2562c0c967efe11597e61c5e748d659b0e95128513c625e20d46R160-R164) for you

Reply:
Add a draft yesterday but I have evolved it further whilst exploring the proposed refactor to `answers`.
https://github.com/DFE-Digital/early-years-foundation-recovery/pull/566/commits/4dacf2ed78d00237e1607d003e7303890644bad5